checksec

```diff
@@ -780,6 +791,22 @@ kernelcheck() {
     echo_message "\033[31mDisabled\033[m\n" "Disabled," " protect_hardlinks='no'" '"protect_hardlinks":"no",'
   fi
 
+  echo_message "  Protected fifos:                        " "" "" ""
```

Comment:
I need to move all of them over to it but if you could wrap these in an outer if to see if fs.protected_fifos is even a possibility.    I am hoping to rework all the checks in the near future like 
https://github.com/slimm609/checksec.sh/blob/master/checksec#L825

so that if (for example) fs.protected_fifos isn't even an option on that version of the kernel, then it will not notify that it is disabled.   The current version checks some functions but not all.